perf/benchmark/flame/README.md

```diff
@@ -56,7 +56,7 @@ Flame graphs are created from data collected using linux `perf_events` by the `p
 1. Run [`get_proxy_perf.sh`](get_proxy_perf.sh) to get the profiling svg. The following command collects samples at `177Hz` for `20s`. The svg file should be created under `flameoutput` dir
```

Comment:
Is this still recording at 177hz? `-s 177` was changed to `-f 99`

Reply:
Done